screening/bahmni.json

```diff
@@ -0,0 +1,139 @@
+      "Medical Standards Supported: SNOMED, LOINC, ICD-10, CIEL",
+      "Standards - HTML/CSS/JS based frontends talk over RESTful APIs. In addition Bahmni also provides HL7 FHIR R4 APIs.  Communication/Notifications use W3 ATOM syndication standard."
+    ],
+    "evidenceStandardSupport": [
+      ""
+    ],
+    "implementBestPractices": "Yes",
+    "bestPracticesList": [
+      "Principles for digital development",
+      "Open source principles, that is, peer production with products such as source code, blueprints, and documentation freely available to the public.",
+      "AGPL guidelines",
+      "Standard Application Security Best Practices."
+    ]
+  },
+  "doNoHarm": {
+    "preventHarm": {
+      "stepsToPreventHarm": "Yes",
+      "additionalInfoMechanismProcessesPolicies": "The use cases in which this product is implemented has not indicated any harmful intentions thus far."
+    },
+    "dataPrivacySecurity": {
+      "collectsPII": "Yes",
+      "typesOfDataCollected": [
+        "Name",
+        "Address",
+        "Date of birth",
+        "Contact details",
+        "Other unique identifiers."
+      ],
+      "thirdPartyDataSharing": "Yes",
+      "dataSharingCircumstances": [
+        "Integration of product with 3rd party is optional and configurable; which provides users of the product to define what they wish to share."
+      ],
+      "ensurePrivacySecurity": "Yes",
+      "privacySecurityDescription": "Bahmni has features like Audit Log to protect and record who has accessed, recorded or modified what health records and also authentication/authorisation features to only allow approved users to access patient data. The Bahmni development team does not have any access to real production data/systems, and those are owned by the organisation that deploys/supports Bahmni, as Bahmni is an on-site deployed product. More documentation on this topic here: https://bahmni.atlassian.net/wiki/spaces/BAH/pages/41713684/Security+Roles+and+Privileges"
+    },
+    "inappropriateIllegalContent": {
+      "collectStoreDistribute": "No",
+      "type": "",
+      "contentFilter": "",
+      "policyGuidelinesDocumentationLink": "",
+      "illegalContentDetection": "",
+      "illegalContentDetectionMechanism": ""
+    },
+    "protectionFromHarassment": {
+      "userInteraction": "Yes",
+      "addressSafetySecurityUnderageUsers": "Yes",
+      "stepsAddressRiskPreventSafetyUnderageUsers": [
+        "The software is not intended for underage users and is expected to be used by professionals in clinical settings."
+      ],
+      "griefAbuseHarassmentProtection": "No",
+      "harassmentProtectionSteps": [
+        "The product does not indicate any harmful intentions in its vanilla state and hence does not restrict the users from accessing."
+      ]
+    }
+  },
+  "locations": {
+    "developmentCountries": [
+      "OpenSource :Across the world",
```

Comment:
I would standardize this, and have one entry for USA, one entry India, as the countries where the majority of contributions are coming from; so that we can later display it easily on a map.

Reply:
Resolved in commit a07f0f7.